integrations/install.sh

```diff
@@ -17,68 +17,68 @@ function show {
 	echo -e "${fnt}${txt}`tput sgr0`"
 }
 
+# checking Prerequisites
+function goPrerequisites {
+    which go
+    if [ $? -ne 0 ]; then
+    show "Need to install Go"
+    exit
+    fi
 
-# Instal Docker
-show "cheking Docker.."
-docker -v
-if [ $? -ne 0 ]; then
-show "Need to install Docker"
-show "do you want install?\n[1]Yes\n[2]No"
-    read -p "> " INPSEL
-	case $INPSEL in
-	    "1")
-            show "### Installing docker  ####"
-            ./install_docker_ubuntu.sh
-			;;
-	    "2")
-            exit
-			;;
-	esac
-fi
-show "done"
+    goV=`go version | { read _ _ v _; echo ${v#go}; }`
+    echo $goV
+    if [[ "$goV" < "1.19.0" ]]; then
+      echo "supported on golang 1.19+"
+      exit
+    fi
+}
+
+function gitPrerequisites {
+        which git
+        if [ $? -ne 0 ]; then
+        show "Need to install Git"
+        exit
+        fi
+}
 
-# Install docker-compose
-show "docker-compose.."
-docker-compose version
-if [ $? -ne 0 ]; then
-show "Need to install docker-compose"
-show "do you want install?\n[1]Yes\n[2]No"
-    read -p "> " INPSEL
-	case $INPSEL in
-	    "1")
-            show "### Installing docker-compose  ####"
-            sudo curl -L "https://github.com/docker/compose/releases/download/1.28.5/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
-            sudo chmod +x /usr/local/bin/docker-compose
-            sudo docker plugin install grafana/loki-docker-driver:latest --alias loki --grant-all-permissions
-			;;
-	    "2")
-            exit
-			;;
-	esac
-fi
-show "done"
+function dockerPrerequisites {
+      which docker
+      if [ $? -ne 0 ]; then
+      show "Need to install Docker"
+      exit
+      fi
+
+      docker -v
+}
 
+function dockerComposePrerequisites {
+      which docker-compose
+      if [ $? -ne 0 ]; then
+      show "Need to install docker-compose"
+      exit
+      fi
+
+      docker-compose version
+}
+
+function mongoDBPrerequisites {
+      which mongo
+      if [ $? -ne 0 ]; then
+      show "Need to install mongo shell"
```

Comment:
not a must, so
"for best experience, we advise you also install Mongo db tools" (+link)